src/main/java/io/cdap/plugin/gcp/dataplex/common/util/DataplexConstants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```

Comment:
nit: 2022
here and in other newly added files

Reply:
done